packages/docs-reanimated/docs/guides/contributing.mdx

```diff
@@ -0,0 +1,318 @@
+        ├── Common // shared C++ code
+        ├── scripts // JavaScript and bash scripts used in the CI pipeline
+        ├── src // reanimated source code
+        └── plugin // source code of the babel plugin
+```
+
+## Handling open issues
+
+Understanding and reproducing the problem can be a very time consuming task. The great way to help other contributors get up to the speed with solving an issue is providing detailed description and _reproducible_ example. The GitHub already has an template for creating issue, nevertheless we still encounter issues that don't have all the necessary details, like:
+
+- repository that we can clone and quickly see the problem,
+- very generic reproduction steps,
+- missing description,
+- not related or truncated stack trace.
+
+What you can do is ask the owner for those details or try provide them by yourself - try to reproduce the problem and provide missing details, so other developer can work on solving the issue straight away! 🎉
+
+## Helping with documentation
+
+One of many ways you can contribute to this project is by improving the documentation. We try to keep it up to date, but there is still plenty room for improvement. It is also important to update the docs whenever you introduce changes to the API or you discover any inconsistency in the current API or behavior. This is also one of the most straightforward ways to contribute. Some of the page are still in old, v2 format. You can help us convert them to th Reanimated 3 structure.
+
+The Reanimated documentation is using Docusaurus and hosted on GitHub Pages. The easiest way to start adding your changes it by clicking **Edit this page** located at the bottom of most pages.
+
+### Editing source locally
+
+If you want to contribute more complicated changes, it's a good idea to edit locally. That way you can run the changes locally to make sure everything will look as expected to the end users. To get started, follow the steps below:
+
+1. Read [How to Open a Pull Request](https://reactnative.dev/contributing/how-to-open-a-pull-request)
+2. Clone [react-native-reanimated](https://github.com/software-mansion/react-native-reanimated/)
+3. `cd react-native-reanimated`
+4. `yarn`
+5. `cd packages/docs-reanimated`
+6. `yarn && yarn start`
+
+This will start your local Docusaurus server running the documentation.
+
+### Project structure
+
+Here you can find the most important parts of the `react-native-reanimated/packages/docs-reanimated` project. Not all folders are listed for brevity.
+
+```
+├── docs // contains folder with md/mdx files, each folder is a section in the side navigation, and each file is documentation page
+├── src
+│   ├── components // contains components used to make the pages interactive
+│   └── examples // contains examples that are later presented in the documentation
+└── static
+    ├── img // contains images used in the docs
+    └── recordings // contains video recordings of the examples
+```
```

Comment:
Looks a bit strange when scrolled with these buttons over it - they are kinda messing with each other. (cc @patrycjakalinska @kacperkapusciak )
 
https://github.com/software-mansion/react-native-reanimated/assets/77503811/0f4deebe-8c1e-4c8e-89c2-a826f7308030

Reply:
You could also make the sentence shorter - I think it would do the job

Reply:
i prefer to have it as a tree structure 

yes, you can add a prop but I don't think these buttons are a real issue tho

Reply:
Ok, so I'm leaving as it is